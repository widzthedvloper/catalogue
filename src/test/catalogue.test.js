import React from 'react';
import '@testing-library/jest-dom';
import renderer from 'react-test-renderer';

it('renders span with text correctly', () => {
    const tree = renderer
    .create(<span>CATALOGUE RECEIPES</span>)
    .toJSON();
    expect(tree).toMatchSnapshot();
})

it('renders span with text correctly', () => {
    const tree = renderer
    .create(<h3>Bakewell tart</h3>)
    .toJSON();
    expect(tree).toMatchSnapshot();
})

it('renders span with text correctly', () => {
    const tree = renderer
    .create(<h3>Bread and Butter Puddind</h3>)
    .toJSON();
    expect(tree).toMatchSnapshot();
})

it('renders span with text correctly', () => {
    const tree = renderer
    .create(<h3>Beef Wellington</h3>)
    .toJSON();
    expect(tree).toMatchSnapshot();
})

it('renders span with text correctly', () => {
    const tree = renderer
    .create(<h3>Baingan Bharta</h3>)
    .toJSON();
    expect(tree).toMatchSnapshot();
})

it('renders span with text correctly', () => {
    const tree = renderer
    .create(<h3>Beef Brisket Pot Roast</h3>)
    .toJSON();
    expect(tree).toMatchSnapshot();
})

it('renders span with text correctly', () => {
    const tree = renderer
    .create(<h3>Beef Sunday Roast</h3>)
    .toJSON();
    expect(tree).toMatchSnapshot();
})

it('renders span with text correctly', () => {
    const tree = renderer
    .create(<h3>Braised Beef Chilli</h3>)
    .toJSON();
    expect(tree).toMatchSnapshot();
})

it('renders span with text correctly', () => {
    const tree = renderer
    .create(<h3>Beef stroganoff</h3>)
    .toJSON();
    expect(tree).toMatchSnapshot();
})

it('renders span with text correctly', () => {
    const tree = renderer
    .create(<h3>Broccoli & Stilton soup</h3>)
    .toJSON();
    expect(tree).toMatchSnapshot();
})

it('renders span with text correctly', () => {
    const tree = renderer
    .create(<h3>Bean & Sausage Hotpot</h3>)
    .toJSON();
    expect(tree).toMatchSnapshot();
})

it('renders span with text correctly', () => {
    const tree = renderer
    .create(<h3>Banana Pancakes</h3>)
    .toJSON();
    expect(tree).toMatchSnapshot();
})

it('renders span with text correctly', () => {
    const tree = renderer
    .create(<h3>Beef Dumpling Stew</h3>)
    .toJSON();
    expect(tree).toMatchSnapshot();
})

it('renders span with text correctly', () => {
    const tree = renderer
    .create(<h3>Beef and Mustard Pie</h3>)
    .toJSON();
    expect(tree).toMatchSnapshot();
})
