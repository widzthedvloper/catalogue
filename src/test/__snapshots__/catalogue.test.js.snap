// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders span with text correctly 1`] = `
<span>
  CATALOGUE RECEIPES
</span>
`;

exports[`renders span with text correctly 2`] = `
<h3>
  Bakewell tart
</h3>
`;

exports[`renders span with text correctly 3`] = `
<h3>
  Bread and Butter Puddind
</h3>
`;

exports[`renders span with text correctly 4`] = `
<h3>
  Beef Wellington
</h3>
`;

exports[`renders span with text correctly 5`] = `
<h3>
  Baingan Bharta
</h3>
`;

exports[`renders span with text correctly 6`] = `
<h3>
  Beef Brisket Pot Roast
</h3>
`;

exports[`renders span with text correctly 7`] = `
<h3>
  Beef Sunday Roast
</h3>
`;

exports[`renders span with text correctly 8`] = `
<h3>
  Braised Beef Chilli
</h3>
`;

exports[`renders span with text correctly 9`] = `
<h3>
  Beef stroganoff
</h3>
`;

exports[`renders span with text correctly 10`] = `
<h3>
  Broccoli & Stilton soup
</h3>
`;

exports[`renders span with text correctly 11`] = `
<h3>
  Bean & Sausage Hotpot
</h3>
`;

exports[`renders span with text correctly 12`] = `
<h3>
  Banana Pancakes
</h3>
`;

exports[`renders span with text correctly 13`] = `
<h3>
  Beef Dumpling Stew
</h3>
`;

exports[`renders span with text correctly 14`] = `
<h3>
  Beef and Mustard Pie
</h3>
`;
